js/lib/bootbox.js

```diff
@@ -0,0 +1,985 @@
+/**
```

Comment:
Agreed.  As he noted, @aeschylus is changing the build process, so he will incorporate that with the rest of his work.